data/json/mutations/mutations.json

```diff
@@ -339,25 +339,26 @@
     "name": { "str": "Weak Photophore" },
     "points": 1,
     "description": "A photophore has grown from your head; you can make it glow softly.  This will make you very visible in the dark, ideal for attracting a partner during mating season.",
-    "active": true,
     "cost": 9,
     "time": "225 h",
     "kcal": true,
     "encumbrance_covered": [ [ "head", 5 ] ],
     "types": [ "BIOLUM" ],
     "prereqs": [ "BIOLUM0" ],
     "changes_to": [ "BIOLUM2" ],
-    "transform": { "target": "BIOLUM1_active", "msg_transform": "Your photophore starts glowing.", "active": true, "moves": 10 },
-    "category": [ "ELFA", "INSECT", "FISH" ]
+    "category": [ "ELFA", "INSECT", "FISH" ],
+    "active": true,
+    "activated_eocs": [ "EOC_BIOLUM1" ]
   },
   {
     "type": "mutation",
     "id": "BIOLUM1_active",
     "name": { "str": "Weak Photophore (on)" },
     "description": "Your photophore is glowing softly.",
-    "copy-from": "BIOLUM1",
```

Comment:
This effectively erases the `BIOLUM1` mutation from any existing saves which had it acitve.

Reply:
Hm. The only thing I can think of is obsoleting BIOLUM1_active, adding an EOC that gives you BIOLUM1 and gets rid of BIOLUM1_active if you have it, and then moving the active from of BIOLUM1 to a new ID. But that seems kinda hacky and I don't think it'd get merged. Any thoughts on a better method?

Reply:
You should just be able to migrate the mutation `BIOLUM1_ACTIVE` to `BIOLUM1`, the conditional enchantment will re-check on load (when the enchantment cache is dirtied, which happens many times on load) if it is necessary and apply it when the enchantment's conditions are fulfilled.

Reply:
Oh right, I forgot about migrations. Thanks for the help, I'll reopen when I get that set up.